.github/workflows/ci.yml

```
@@ -13,38 +10,23 @@ permissions:
+      - uses: prefix-dev/setup-pixi@v0.8.1
+        with:
+          cache: true
+          frozen: true
+          environments: ${{ matrix.environment }}
+      - name: CI
+        run: |
+          pixi run -e ${{ matrix.environment }} ci 
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)